db/sqldb/portfolio.go

```diff
@@ -182,7 +183,7 @@ func (d *DB) DeletePortfolio(tx db.Tx, id pacta.PortfolioID) ([]pacta.BlobURI, e
 
 func rowToPortfolio(row rowScanner) (*pacta.Portfolio, error) {
 	p := &pacta.Portfolio{Owner: &pacta.Owner{}, Blob: &pacta.Blob{}}
-	hd := pgtype.Timestamptz{}
+	props := []byte{}
```

Comment:
There's no need to manually encode/decode all of these things, `JSON` and `JSONB` types can be (un)marshalled the same way as normal JSON things, see https://pkg.go.dev/github.com/jackc/pgx/v5/pgtype#hdr-JSON_Support

Reply:
TIL!!! Thank you!

Reply:
This is a gorgeous simplification BTW